src/views/TradingCompetition/components/CompetitionHeadingText.tsx

```diff
@@ -22,7 +22,6 @@ const H2SizeStyles = (theme: DefaultTheme) => `
 `
 
 const sharedStyles = (props: HeadingProps) => `
-  color: ${props.textColor ? props.textColor : '#ffffff'};
```

Comment:
We still need this color to be assigned, but the prop is OK to be removed, so just:

```css
color: #ffffff;
```

if removed entirely:
<img width="434" alt="Screenshot 2021-04-09 at 15 02 08" src="https://user-images.githubusercontent.com/79279477/114191781-b1845e00-9944-11eb-968e-25abeca0c59d.png">

Reply:
Done